<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1824</width>
    <height>984</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>211</red>
        <green>215</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>238</green>
        <blue>236</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>87</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>211</red>
        <green>215</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>238</green>
        <blue>236</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>87</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>211</red>
        <green>215</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>87</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>87</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>epiInfo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/ressources/logo_hub.png</normaloff>:/ressources/logo_hub.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>76</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>76</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>76</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>22</red>
         <green>54</green>
         <blue>76</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QLabel" name="contribute">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>570</y>
      <width>671</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Contribuez au projet sur&lt;br/&gt;github.com/hub-marseille/epiInfo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tram">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>780</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Horaires trams</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="metro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>780</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Horaires m√©tros</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bus">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>780</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Horaires bus</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>20</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>45</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>xx:xx:xx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1824</width>
      <height>984</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/ressources/background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="information">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>770</y>
      <width>861</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Aucune information pour le moment.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="tram_hour">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>780</y>
      <width>111</width>
      <height>177</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open 24 Display St</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Heure tram</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="metro_hour">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>780</y>
      <width>121</width>
      <height>177</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open 24 Display St</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Heure Metro</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bus_hour">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>780</y>
      <width>111</width>
      <height>177</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open 24 Display St</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Heure Bus</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tram_error">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>820</y>
      <width>481</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tram Error</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="metro_error">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>820</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Metro Error</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bus_error">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>820</y>
      <width>481</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bus Error</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gui_room">
    <property name="geometry">
     <rect>
      <x>1128</x>
      <y>84</y>
      <width>150</width>
      <height>228</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="guinness">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>60</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Guiness</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scifi">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>60</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scifi</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cousteau">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>60</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cousteau</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="retro">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>370</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Retro</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hub">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>360</y>
      <width>192</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HUB</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sci_room">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>84</y>
      <width>210</width>
      <height>228</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cou_room">
    <property name="geometry">
     <rect>
      <x>1493</x>
      <y>84</y>
      <width>204</width>
      <height>228</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="retro_room">
    <property name="geometry">
     <rect>
      <x>917</x>
      <y>390</y>
      <width>441</width>
      <height>224</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hub_room">
    <property name="geometry">
     <rect>
      <x>1362</x>
      <y>390</y>
      <width>445</width>
      <height>224</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1545</x>
      <y>530</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/ressources/logo_hub.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Copyright">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>910</y>
      <width>521</width>
      <height>91</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Made by Hub Team with Diane GUILLOT &amp; Thibaut DAVID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="next_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>801</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Opificio Neue</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Planning et affichage des activit√©s √† venir&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>contribute</zorder>
   <zorder>tram</zorder>
   <zorder>time</zorder>
   <zorder>information</zorder>
   <zorder>tram_hour</zorder>
   <zorder>tram_error</zorder>
   <zorder>gui_room</zorder>
   <zorder>guinness</zorder>
   <zorder>sci_room</zorder>
   <zorder>scifi</zorder>
   <zorder>cou_room</zorder>
   <zorder>cousteau</zorder>
   <zorder>retro_room</zorder>
   <zorder>retro</zorder>
   <zorder>hub_room</zorder>
   <zorder>hub</zorder>
   <zorder>label</zorder>
   <zorder>Copyright</zorder>
   <zorder>metro</zorder>
   <zorder>bus_error</zorder>
   <zorder>metro_error</zorder>
   <zorder>metro_hour</zorder>
   <zorder>next_2</zorder>
   <zorder>bus_hour</zorder>
   <zorder>bus</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
